### ISSUE TITLE
Phase out mentions of contribute.json?

### ISSUE THREAD

**Issue by janbrasna** [mozilla/infosec.mozilla.org#186]
This month www.contributejson.org was taken down and the repo archived, and the scoring was also removed from Observatory:
- https://github.com/mozilla/http-observatory/pull/526
- https://github.com/mozilla/http-observatory-website/pull/308

I'm not sure it's worth keeping a now unmaintained "standard" as a prominent feature in Web Security guidelines.

**Comment by gene1wood** [comment]
I agree. If you want to PR removal of the `contribute.json` section, please do so.
